### PR TITLE
Separate the mymove repo from the ppp-infra repo

### PR DESCRIPTION
**Pull request by chrisgilmerproj**
## Description

Folks don't often update their ppp-infra checkout which causes issues when we want to update shared tooling. Making this change separates out the repos completely. Folks will still need to use the ppp-infra repo to rotate their aws keys or for setting up a new account. Otherwise there's no need for them to be tied together.

This is a follow on to the upgrading of aws-vault in https://github.com/transcom/ppp-infra/pull/641

## Setup

Try it out!

```sh
rm bin/chamber
rm $GOPATH/bin/chamber
prereqs
cp .envrc.chamber.template .envrc.chamber
direnv allow
```

### CODE REVIEW

**Comment by chrisgilmerproj** [comment]
Currently this is going to require that folks update their `.envrc.chamber` file if they have one. So I'm going to add some logic to ensure that its a more seemless transition.

**Comment by tinyels** [comment]
When I run `chamber read app-devlocal cloud_front_private_key'` I get this response:

`aws-vault: error: unknown long flag '--duration', try --help`

File: README.md [hunk 1]
```diff
@@ -217,7 +217,8 @@ For managing local environment variables, we're using [direnv](https://direnv.ne
 
 Run `direnv allow` to load up the `.envrc` file. It should complain that you have missing variables which you will rectify in one of the following ways.
 
-You can add a `.envrc.local` file. One way to do this is using the [chamber tool](https://github.com/segmentio/chamber) to read secrets from AWS vault.  You must have the infra-com repo already cloned and must add the path to it in your .envrc.local (`PPP_INFRA_PATH='~/yourlocalpath/ppp-infra'`) Then run `chamber env app-devlocal >> .envrc.local`. If you don't have access to chamber you can also `touch .envrc.local` and add any values that the output from direnv asks you to define. Instructions are in the error messages.
+You can add a `.envrc.local` file. One way to do this is using the [chamber tool](https://github.com/segmentio/chamber) to read secrets from AWS vault.
+Then run `chamber env app-devlocal >> .envrc.local`. If you don't have access to chamber you can also `touch .envrc.local` and add any values that the output from direnv asks you to define. Instructions are in the error messages.
```
Comment (tinyels) [inline README.md]: When I run `chamber env app-devlocal` I get this error:

```
Error: unknown command "env" for "chamber"
Run 'chamber --help' for usage.
```

and `env` isn't one of the options shown in help.

File: .envrc.chamber.template [hunk 2]
```diff
@@ -9,9 +9,9 @@
 
 # This line grabs any available secrets in app-devlocal and adds them to the environment.
 # Lines that begin with `require ...` should still run to verify the secret is set.
-if ! AWS_VAULT_KEYCHAIN_NAME=login aws-vault exec transcom-ppp -- chamber list app-devlocal --retries=1 > /dev/null ; then
+if ! chamber list app-devlocal --retries=1 > /dev/null ; then
   log_error "Unable to access app-devlocal variables with chamber."
   log_error "Login to chamber with 'chamber list app-devlocal'."
 else
-  eval "$(AWS_VAULT_KEYCHAIN_NAME=login aws-vault exec transcom-ppp -- chamber env app-devlocal --retries=1)"
+  eval "$(chamber env app-devlocal --retries=1)"
```
Comment (tinyels) [inline .envrc.chamber.template]: This line fails for me (even in bash) 
``` direnv: loading .envrc
direnv: loading .envrc.chamber
Error: unknown command "env" for "chamber"```

Comment (chrisgilmerproj) [reply .envrc.chamber.template]: That seems rather odd. Try running `chamber` and see if the output looks like this:

```
chamber
CLI for storing secrets

Usage:
  chamber [command]

Available Commands:
  delete        Delete a secret, including all versions
  env           Print the secrets from the parameter store in a format to export as environment variables
  exec          Executes a command with secrets loaded into the environment
  export        Exports parameters in the specified format
  find          Find the given secret across all services
  help          Help about any command
  history       View the history of a secret
  import        import secrets from json
  list          List the secrets set for a service
  list-services List services
  read          Read a specific secret from the parameter store
  version       print version
  write         write a secret

Flags:
  -b, --backend string             Backend to use; AKA $CHAMBER_SECRET_BACKEND
                                        null: no-op
                                        ssm: SSM Parameter Store
                                        s3: S3; requires --backend-s3-bucket
                                        s3-kms: S3 using AWS-KMS encryption; requires --backend-s3-bucket and --kms-key-alias set (if you want to write or delete keys). (default "ssm")
      --backend-s3-bucket string   bucket for S3 backend; AKA $CHAMBER_S3_BUCKET
  -h, --help                       help for chamber
      --kms-key-alias string       KMS Key Alias for writing and deleting secrets; AKA $CHAMBER_KMS_KEY_ALIAS. This option is currently only supported for the S3-KMS backend. (default "alias/parameter_store_key")
  -r, --retries int                For SSM, the number of retries we'll make before giving up (default 10)
      --verbose                    Print more information to STDOUT

Use "chamber [command] --help" for more information about a command.
```

`chamber env` has been a valid command for a long time.

Also, make sure you delete `bin/chamber`.

File: README.md (hunk 1 above)
Comment (chrisgilmerproj) [reply README.md]: There is something wrong with your version of `chamber`.

**Pull request merged**